### PR TITLE
defaulting to mainnet for portis wallet, finding networkId from porti…

### PR DESCRIPTION
**Pull request by schultztimothy**
##### Description

It looks like, when using portis wallet, there was an issue finding the networkId which is used to filter out tokens on the checkout page. 

I also was unable to switch the portis wallet to any network beside rinkeby. While looking through the docs it looks like they are expecting a network parameter https://docs.portis.io/#/quick-start. 

##### Refers/Fixes

fixes: #10429

### CODE REVIEW

**Review by philipjonsen — APPROVED** [review]

**Review by thelostone-mc — COMMENTED** [review]
lgtm. small note:  could we remove the log and commented out code

**Comment by schultztimothy** [comment]
> lgtm. small note: could we remove the log and commented out code

Ahh thank you for catching that 🙌

**Review by thelostone-mc — APPROVED** [review]

File: app/assets/v2/js/cart.js [hunk 1]
```diff
@@ -1131,6 +1134,7 @@ Vue.component('grants-cart', {
         const tokenName = selectedTokens[i];
         const tokenDetails = this.getTokenByName(tokenName, isPolygon);
 
+        console.log({ tokenDetails });
```
Comment (chibie) [inline app/assets/v2/js/cart.js:1137]: can we take out this log?

**Review by chibie — CHANGES_REQUESTED** [review]

File: app/assets/v2/js/cart.js (hunk 1 above)
Comment (thelostone-mc) [reply app/assets/v2/js/cart.js:1137]:
```suggestion
```

**Review by chibie — APPROVED** [review]

**Pull request merged**